SPECS/openssl/openssl-1.1.1-load-default-engines.patch

```diff
@@ -1,8 +1,8 @@
 diff --git a/crypto/init.c b/crypto/init.c
```

Comment:
So this patch is essentially the state after applying https://github.com/microsoft/CBL-Mariner/pull/3182 and https://github.com/microsoft/CBL-Mariner/pull/3368 ?

Reply:
Yes, if nothing else changed there in `main`. I've taken the contents from latest `main` to make sure we've got the same contents in it.